lib/sen_tweet_web/live/stats_live.html.leex

```diff
@@ -1,172 +1,184 @@
 
-<%= for {stream, %{"hourly" => hourly, "daily" => daily}} <- @streams do %>
-
-<section class="hero is-light">
-  <div class="hero-body">
-    <div class="container">
-      <h1 class="title is-1 is-spaced has-text-centered">
-        Hourly Stats
-      </h1>
-
-      <nav class="level">
-        <div class="level-item has-text-centered">
-          <div>
-            <p class="heading">Stream</p>
-            <p class="title"><%= String.upcase(stream) %></p>
+<%= if Enum.count(@streams) > 0 do %>
```

Comment:
You can pattern match and avoid the count here.
```
if @streams == [] do
  # no streams
else
  # show stats
```

Reply:
The `match?/2` macro is an alternative to the above. Whatever reads better to you.
```
if match?([], @streams) do
```